recipe/meta.yaml

```diff
@@ -109,3 +113,4 @@ extra:
     - mingwandroid
     - msarahan
     - xylar
+    - dopplershift
```

Comment:
I invited you to conda-forge so you would be here. My mission is complete! I will go back into the void now :smile:

Reply:
😆